lib/ex_unit/lib/ex_unit/case.ex

```diff
@@ -578,7 +578,7 @@ defmodule ExUnit.Case do
   display. You can use `ExUnit.plural_rule/2` to set a custom
   pluralization.
   """
-  @doc since: "1.10.0"
+  @doc since: "1.11.0"
```

Comment:
I initially looked through this file's history, version tag by version tag to check this. It would have been faster had I just grepped the changelogs :)

https://github.com/elixir-lang/elixir/blob/v1.11/CHANGELOG.md#exunit-1